testdata/expected/program/java-android-gradle-sample/default-group/java-android-gradle-sample/JavaArtifact.unit.json

```diff
@@ -0,0 +1 @@
+{"Name":"default-group/java-android-gradle-sample","Type":"JavaArtifact","Repo":"github.com/alexsaveliev/java-android-gradle-sample","Files":["build/generated/source/buildConfig/debug/de/goddchen/android/gradle/helloworld/BuildConfig.java","build/generated/source/r/debug/de/goddchen/android/gradle/helloworld/R.java","src/main/java/de/goddchen/android/gradle/helloworld/MainActivity.java"],"Data":{"Android":"21","BootClassPath":["sdk/platforms/android-21/android.jar"],"ClassPath":["build/intermediates/dependency-cache/debug","build/intermediates/classes/debug"],"Description":"","GradleFile":"build.gradle","SourcePath":[["/java-android-gradle-sample","unspecified","src/main/java"],["/java-android-gradle-sample","unspecified","src/test/java"],["default-group/java-android-gradle-sample","unspecified","build/intermediates/classes/debug"],["default-group/java-android-gradle-sample","unspecified","build/intermediates/dependency-cache/debug"]],"SourceVersion":"1.8"}}
```

Comment:
Can we update this to point to `github.com/sgtest/java-android-gradle-sample` instead of `github.com/alexsaveliev/java-android-gradle-sample`. It would just be nice to have all the test cases in one place :)